src/pages/Register/Register.tsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import { useRegisterQuery } from './api/useRegisterQuery';
+import { TextField, Button, Box, Grid, Typography } from '@mui/material';
+
+type FormData = {
+  username: string;
+  name: string;
+  surname: string;
+  email: string;
+  password: string;
+  repeat_password: string;
```

Comment:
are you sure about usage of `snake_case` instead `camelCase`?

Reply:
we have repeat_password on the backend